Pre-migration CQA task for the REST API title (#4937)
theforeman/foreman-documentation: guides/common/modules/con_introduction-to-project-api.adoc

= Introduction to {Project} API

[role="_abstract"]

> {ProjectName} provides a Representational State Transfer (REST) API that gives you control over your {ProjectName} environment outside of the standard web interface, enabling integration with custom scripts or external applications over HTTP.

Reviewer: {ProjectName} is just {Project}.
Rewrote into shorter sentences, with more user focus.
Avoid saying that something "enables", or "allows" users. That's feature-centric language. See IBM SG.

```suggestion
With the Representational State Transfer (REST) API of {Project} , you can control your {Project} environment outside of the standard web interface. That way, you can integrate your {Project} with custom scripts or external applications over HTTP.
```

> With the Representational State Transfer (REST) API of {Project} , you can control your {Project} environment outside of the standard web interface. That way, you can integrate your {Project} with custom scripts or external applications over HTTP.

Reviewer 2: ```suggestion
With the Representational State Transfer (REST) API of {Project}, you can control your {Project} environment outside of the standard web interface.
That way, you can integrate your {Project} with custom scripts or external applications over HTTP.
```

whitespace & one sentence per line.
